test/spec/Feature/Query/EmbedDisambiguationSpec.hs

```diff
@@ -13,31 +13,6 @@ spec :: SpecWith ((), Application)
 spec =
   describe "resource embedding disambiguation" $ do
     context "ambiguous requests that give 300 Multiple Choices" $ do
-      it "errs when there's a table and view that point to the same fk" $
-        get "/message?select=id,body,sender(name,sent)" `shouldRespondWith`
-          [json|
```

Comment:
This now works(comes from https://github.com/PostgREST/postgrest/pull/1401). I believe ignoring views was the correct solution all along.